spiritual-care.csl

```diff
@@ -0,0 +1,654 @@
+            <text variable="issue" prefix="(" suffix=")"/>
+          </group>
+          <text variable="page"/>
+        </group>
+        <choose>
+          <if variable="issued">
+            <choose>
+              <if match="none" variable="page issue volume">
+                <text variable="DOI" prefix=" doi:"/>
+                <text term="advance-online-publication" prefix=". "/>
+              </if>
+            </choose>
+          </if>
+        </choose>
+      </if>
+      <else-if type="article-newspaper">
+        <group delimiter=" " prefix=", ">
+          <label form="short" variable="page"/>
+          <text variable="page"/>
+        </group>
+      </else-if>
+      <else-if type="book graphic motion_picture report song paper-conference entry-encyclopedia entry-dictionary webpage post-weblog" match="any">
+        <group delimiter=", " prefix=", ">
+          <choose>
+            <if match="any" variable="volume">
+              <group>
+                <text text-case="capitalize-first" suffix=" " term="volume"/>
+                <number form="numeric" variable="volume"/>
+              </group>
+            </if>
+            <else>
+              <group>
+                <text form="short" text-case="capitalize-first" suffix=" " term="volume" plural="true"/>
+                <number form="numeric" variable="number-of-volumes" prefix="1&#8211;"/>
+              </group>
+            </else>
+          </choose>
+        </group>
+      </else-if>
+      <else-if type="chapter" match="any">
+        <group delimiter=". " prefix=". ">
+          <choose>
+            <if match="any" variable="volume">
+              <group>
+                <text term="volume" text-case="capitalize-first" suffix=" "/>
+                <number form="numeric" variable="volume"/>
+              </group>
+            </if>
+            <else>
+              <group>
+                <text text-case="capitalize-first" term="volume" plural="true"/>
+                <number form="numeric" variable="number-of-volumes"/>
+              </group>
+            </else>
+          </choose>
+          <group>
+            <text variable="page" prefix=" "/>
+          </group>
+        </group>
+      </else-if>
+      <else-if type="legal_case">
+        <group delimiter=" " prefix=" (" suffix=")">
+          <text variable="authority"/>
+          <choose>
+            <if match="any" variable="container-title">
+              <date form="numeric" variable="issued" date-parts="year"/>
+            </if>
+            <else>
+              <date form="text" variable="issued"/>
+            </else>
+          </choose>
+        </group>
+      </else-if>
+      <else-if type="bill legislation" match="any">
+        <date variable="issued" prefix=" (" suffix=")">
+          <date-part name="year"/>
+        </date>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <group>
+      <choose>
```

Comment:
Delete lines 520-524.
That is solved by the group delimiter in line 620 and will give you issues.